Simplenote/NoteEditorViewController.m

```diff
@@ -588,9 +588,8 @@ - (void)menuWillOpen:(NSMenu *)menu
     
     [self updateCounts];
     pinnedItem.state = [self selectedNotesPinned] ? NSOnState : NSOffState;
+    markdownItem.state = [self selectedNotesMarkdowned] ? NSOnState : NSOffState;
     [collaborateItem setEnabled:numSelectedNotes == 1];
-    [publishItem setEnabled:numSelectedNotes == 1];
```

Comment:
Should these lines have been removed? The publish icon is still enabled now if I select multiple notes, but doesn't do anything if I click it. And the history popup shows 'version: null' if I click it with multiple notes enabled.

Reply:
Nope! Not sure how that happened :) Nice catch.